### PR TITLE
Changes based on feedback and Bug fixes

### PR DESCRIPTION
**Pull request by tonymontaro**
#### What does this PR do?
This PR adds changes made to the application based on feedback and fixes known bugs

#### Description of task performed
- update the 'continue learning' section of the homepage
- add bottom border to active links on the navbar
- update the certification page
- implement share link functionality on assessment page
- update the homepage
- update the tools page
- update social sharing links
- update the feedback page

#### How to test?
- Visit: https://codeforafrica-coursebuilder.appspot.com/

### CODE REVIEW

File: coursebuilder/modules/explorer/_static/components/pages/feedback-page.html [hunk 1]
```diff
@@ -7,56 +7,53 @@
...
                   </ul>
                 </div>
-                <h3 style="font-size: 1.375em; margin: 1.9em 0 0.9em; color: #4054B2">
+                <h3>
                   General feedback
                 </h3>
 
                 <div class="row white">
-                  <form class="col s12">
+                  <form class="col s12" style="">
```
Comment (DavidLemayian) [inline coursebuilder/modules/explorer/_static/components/pages/feedback-page.html]: Unused style tag.

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html [hunk 2]
```diff
@@ -73,85 +73,72 @@ <h2 class="section-label">An award winning platform</h2>
                 </div>
             </div>
 
-            <!--  How it works  -->
-            <div class="section how-it-works">
-                <div class="carousel-section-wrapper">
-                    <h2 class="section-label">How It Works</h2>
+          <!--  How it works  -->
+  <div class="section how-it-works">
```
Comment (DavidLemayian) [inline coursebuilder/modules/explorer/_static/components/pages/homepage.html]: Is the indentation correct?

File: frontend/scss/pages/_certification.scss [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    display: block;
+    max-width: $context-width;
+    margin: 0 auto;
+  }
+  
+  @include thin-paragraph;
+}
+
+
+}
```
Comment (DavidLemayian) [inline frontend/scss/pages/_certification.scss]: What's the extra curly braces for?

File: frontend/scss/pages/_feedback.scss [hunk 4]
```diff
@@ -0,0 +1,164 @@
+#feedback
+{
+background-color: #f5f5f5;
+#email
+{
```
Comment (DavidLemayian) [inline frontend/scss/pages/_feedback.scss]: Indentation but also the opening curly brace should come after the element?

File: frontend/scss/pages/_partners.scss [hunk 5]
```diff
@@ -35,7 +39,10 @@
   .blue-section-inner h2 {
     margin-bottom: 0;
   }
-
+  .row{
```
Comment (DavidLemayian) [inline frontend/scss/pages/_partners.scss]: Space after the element?

**Review by DavidLemayian — CHANGES_REQUESTED** [review]
I like that we are placing the CSS in stylesheets but there are still a couple of in-line style edits that we should fix.

Also, is everyone using a linter for their code? I can see some of the edits don't follow any style guide and indentation is all over the place.

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html (hunk 2 above)
Comment (tonymontaro) [reply coursebuilder/modules/explorer/_static/components/pages/homepage.html]: @DavidLemayian All files have been properly formatted. Python files are indented with 4 spaces (as recommend by pep 8) while html, css and javascript files are indented with 2 spaces.

File: .gitignore [hunk 6]
```diff
@@ -2,3 +2,4 @@
 **/*.idea
 **/*node_modules
 frontend/jinja
+package-lock.json
```
Comment (DavidLemayian) [inline .gitignore]: This should be committed into source control - https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

File: coursebuilder/modules/explorer/_static/components/pages/feedback-page.html [hunk 7]
```diff
@@ -7,166 +7,180 @@
...
-                         style="color: #4054b2; font-weight: 700; font-family: Roboto,Arial,sans-serif;">SHARE</a>
+                      <a href="https://freeformers.com/" class="right">SHARE</a>
                     </div>
                   </form>
                 </div>
 
-                <div>
-                  <p style="padding: 2rem; line-height: 2.8rem; color: #757575;">
-                    The information you submit will be used in compliance with Google’s <a class="blue-text">Privacy
+                <div class="footer-div">
```
Comment (DavidLemayian) [inline coursebuilder/modules/explorer/_static/components/pages/feedback-page.html]: Should probably be called `footer-privacy`?

File: coursebuilder/modules/explorer/_static/components/pages/feedback-page.html [hunk 8]
```diff
@@ -7,166 +7,180 @@
...
+                  <p>
+                    The information you submit will be used in compliance with Google’s <a>Privacy
                     Policy</a>. We may contact you at the e-mail address you provided.
```
Comment (DavidLemayian) [inline coursebuilder/modules/explorer/_static/components/pages/feedback-page.html]: Hmm.. This footer privacy/policy seems to be repeating. Any way to not have it repeating?

File: coursebuilder/modules/explorer/_static/components/top-bar/top-bar.html [hunk 9]
```diff
@@ -65,13 +65,13 @@
...
+                      <li><a class="nav-link" href="#!homepage">Home</a></li>
+                      <li><a class="nav-link" href="#!explorer">Topic Library</a></li>
+                      <li><a class="nav-link" href="#!certification">Certification</a></li>
+                      <li><a class="nav-link" href="#!events">Events</a></li>
+                      <li><a class="nav-link" href="#!partners">Partners</a></li>
                       <!-- Dropdown Trigger -->
-                      <li><a class="dropdown-button" href="#!" data-activates="dropdown1">More<i
+                      <li><a class="dropdown-button nav-link" href="#!" data-activates="dropdown1">More<i
                               class="material-icons right">arrow_drop_down</i></a>
                       </li>
```
Comment (DavidLemayian) [inline coursebuilder/modules/explorer/_static/components/top-bar/top-bar.html:76]: Aah.. The missing class. :)

File: coursebuilder/views/base.html [hunk 10]
```diff
@@ -235,226 +235,54 @@
...
-              </option>
-
-              <option value="https://learndigital.withgoogle.com/dijitalatolye">
-                Türkçe
+                France
               </option>
 
               <option value="https://learndigital.withgoogle.com/digitalgarage" selected="">
                 United Kingdom
               </option>
```
Comment (DavidLemayian) [inline coursebuilder/views/base.html]: Probably just "Africa" and it points to `https://courses.academy.africa/` for now?

File: frontend/partners.html [hunk 11]
```diff
@@ -131,7 +131,7 @@
            <div class="col s12 m6 l6">
              <div class="card small">
                <div class="card-image">
-                 <img src="images/c4a.png">
+                 <img class="more-hight" src="images/c4a.png">
```
Comment (DavidLemayian) [inline frontend/partners.html]: What is "hight"?

File: frontend/scss/pages/_faq.scss [hunk 12]
```diff
@@ -56,32 +53,40 @@
...
     color: rgba(0, 0, 0, 0.87);
     background-color: white;
   }
-  #h2
-  {
-    padding-left: 50px; border: 30px
+  #h2 {
+    padding-left: 50px;
+    border: 30px
   }
```
Comment (DavidLemayian) [inline frontend/scss/pages/_faq.scss]: Wait, there is an id named `#h2`?? That seems like a bad idea..

**Review by DavidLemayian — CHANGES_REQUESTED** [review]

File: coursebuilder/views/base.html (hunk 10 above)
Comment (tonymontaro) [reply coursebuilder/views/base.html]: @DavidLemayian Those are the languages (to be) supported by the site, see https://trello.com/c/VbYBu4ob/65-footer-on-main-page-home-page

File: frontend/partners.html (hunk 11 above)
Comment (tonymontaro) [reply frontend/partners.html]: It has been given a more descriptive class name.

File: coursebuilder/views/base.html (hunk 10 above)
Comment (DavidLemayian) [reply coursebuilder/views/base.html]: @andela-angene Actually you'll notice from the DigitalGarage site, the options are region not language. If you select a different region, it showcases different partners, content, and yes, language does change as well.

File: coursebuilder/views/student_profile.html [hunk 13]
```diff
@@ -33,31 +32,7 @@
...
-                         widht="22" height="22">
-                    Facebook</a></li>
-                <li>
-                  <a href="https://www.linkedin.com/in/share?url=https://courses.academy.africa">
-                    <img src="/modules/explorer/_static/images/linkedln.png" alt="Linkedln"
-                         widht="22" height="22">
-                    Linkedln</a></li>
-                <li class="divider"></li>
-              </ul>
+              {% include 'share_dropdown.html' %}
```
Comment (DavidLemayian) [inline coursebuilder/views/student_profile.html]: 💯 on going templates instead.

File: coursebuilder/views/base.html (hunk 10 above)
Comment (tonymontaro) [reply coursebuilder/views/base.html]: @DavidLemayian That's true, it's a nice effect. I added 'Africa' as the only option for now.

File: frontend/partners.html (hunk 11 above)
Comment (chegejames) [reply frontend/partners.html]: @andela-angene  We still have "more-hight"?

Comment (tonymontaro) [reply frontend/partners.html]: @chegejames updated.

**Pull request merged**